plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/v1/converter/GlueToTrinoConverter.java

```diff
@@ -156,11 +156,24 @@ public static Table convertTable(com.amazonaws.services.glue.model.Table glueTab
                 (storageDescriptor.isEmpty() && isDeltaLakeTable(tableParameters)) ||
                 (storageDescriptor.isEmpty() && isTrinoMaterializedView(tableType, tableParameters))) {
             // Iceberg tables do not need to read the StorageDescriptor field, but we still need to return dummy properties for compatibility
-            // Delta Lake tables only need to provide a dummy properties if a StorageDescriptor was not explicitly configured.
             // Materialized views do not need to read the StorageDescriptor, but we still need to return dummy properties for compatibility
             tableBuilder.setDataColumns(ImmutableList.of(new Column("dummy", HIVE_INT, Optional.empty(), ImmutableMap.of())));
             tableBuilder.getStorageBuilder().setStorageFormat(HiveStorageFormat.PARQUET.toStorageFormat());
         }
+        else if (isDeltaLakeTable(tableParameters)) {
+            tableBuilder.setDataColumns(ImmutableList.of(new Column("dummy", HIVE_INT, Optional.empty(), ImmutableMap.of())));
+            tableBuilder.setPartitionColumns(ImmutableList.of());
+            if (storageDescriptor.isEmpty()) {
```

Comment:
This should will be handled above in `(storageDescriptor.isEmpty() && isDeltaLakeTable(tableParameters))`, right?
You could remove it from here, or remove from if condition

Reply:
I removed it from the previous condition, thanks for spotting this!